README.md

```diff
@@ -24,6 +24,52 @@ See also:
 
 Jaeger is hosted by the [Cloud Native Computing Foundation](https://cncf.io) (CNCF). If you are a company that wants to help shape the evolution of technologies that are container-packaged, dynamically-scheduled and microservices-oriented, consider joining the CNCF. For details about who's involved and how Jaeger plays a role, read the CNCF [announcement](https://www.cncf.io/blog/2017/09/13/cncf-hosts-jaeger/).
 
+## Features
+
+### High Scalability
+
+Jaeger backend is designed to have no single points of failure and to scale with the business needs.
+For example, any given Jaeger installation at Uber is typically processing several billions of spans per day.
+
+### Native support for OpenTracing
+
+Jaeger backend, Web UI, and instrumentation libraries have been designed from grounds up to support the OpenTracing standard.
+  * Representing traces as directed acyclic graphs (not just trees) via [span references](https://github.com/opentracing/specification/blob/master/specification.md#references-between-spans)
+  * Support for strongly typed span _tags_ and _structured logs_
+  * General distributed context propagation mechanism via _baggage_
+
+### Multiple storage backends
+
+Jaeger supports two popular open source NoSQL databases as trace storage backends: Cassandra 3.4+ and Elasticsearch 5.x/6.x.
+The are ongoing community experiments using other databases, such as ScyllaDB, InfluxDB, Amazon DynamoDB. Jaeger also ships
+with a simple in-memory storage for testing setups.
+
+### Modern Web UI
+
+Jaeger Web UI is implemented in Javascript using popular open source frameworks like React. A number of performance
+improvements have been released in v1.0 to allow the UI to efficiently deal with large volumes of data, and to display
+traces with up to 50,000 spans.
```

Comment:
ive used it to display 80,000 spans. Maybe rephrase as: "tens of thousands of spans" here instead? I'd rather not set a hard upper bound in the docs.